### PR TITLE
docs: add a new branch for 3.5.0

### PR DESCRIPTION
**Pull request by anupriya0703**
Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

**Review by niladrih — APPROVED** [review]
🎉

File: docs/versioned_docs/version-3.4.x/introduction/releases.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ keywords:
 description: This page contains list of supported OpenEBS releases.
 ---
 
-OpenEBS is a collection of data engines and operators to create different types of replicated and local persistent volumes for Kubernetes Stateful workloads. Kubernetes volumes can be provisioned via CSI Drivers or using Out-of-tree Provisioners. The status of the various components as of v3.4.1 are as follows:
+OpenEBS is a collection of data engines and operators to create different types of replicated and local persistent volumes for Kubernetes Stateful workloads. Kubernetes volumes can be provisioned via CSI Drivers or using Out-of-tree Provisioners. The status of the various components as of v3.4.0 are as follows:
```
Comment (avishnu) [inline docs/versioned_docs/version-3.4.x/introduction/releases.md:9]: 3.5.0 ?

File: docs/versioned_docs/version-3.4.x/introduction/releases.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ OpenEBS is a collection of data engines and operators to create different types
   - [Local PV LVM](https://github.com/openebs/lvm-localpv) 1.0.0 (stable)
   - [Local PV Rawfile](https://github.com/openebs/rawfile-localpv) 0.8.0 (beta)
   - [Jiva](https://github.com/openebs/jiva-operator) 3.4.0 (beta) 
-  - [Mayastor](https://github.com/openebs/mayastor) 2.0.1 (stable)
+  - [Mayastor](https://github.com/openebs/mayastor) 2.0.0 (stable)
```
Comment (avishnu) [inline docs/versioned_docs/version-3.4.x/introduction/releases.md:20]: 2.0.1?

**Review by avishnu — APPROVED** [review]
Lgtm

**Pull request merged**